x/chainlink/keeper/msg_server.go

```diff
@@ -15,23 +15,33 @@ var _ types.MsgServer = Keeper{}
 
 const (
 	ErrIncorrectHeightFound = "incorrect height found"
+
+	DataProviderSetChangeTypeAdd          = "Add"
+	DataProviderSetChangeTypeRemove       = "Remove"
+	FeedParamChangeTypeSubmissionCount    = "SubmissionCount"
+	FeedParamChangeTypeHeartbeat          = "Heartbeat"
+	FeedParamChangeTypeDeviationThreshold = "DeviationThreshold"
+	FeedParamChangeTypeCountRewardSchema  = "RewardSchema"
```

Comment:
what does "Count" mean in `FeedParamChangeTypeCountRewardSchema`?

Reply:
```suggestion
	FeedParamChangeTypeRewardSchema  = "RewardSchema"
```

Reply:
sorry, its a typo